docs/evaluation/tutorials/eval_testing_tutorial.mdx

```diff
@@ -0,0 +1,813 @@
+import Tabs from "@theme/Tabs";
+import CodeBlock from "@theme/CodeBlock";
+import {
+  CodeTabs,
+  typescript,
+  python,
+} from "@site/src/components/InstructionsWithCode";
+
+# LangSmith Testing Framework tutorial
+
+This tutorial will show you how to use LangSmith's testing frameworks in both Python and JS/TS to test your LLM applications.
+We will be testing a React agent that answers financial data questions, and will show how to do so in both Python and JS/TS.
```

Comment:
Would also add something like `We will be using LangGraph for orchestration, but this tutorial will work for other frameworks with minor modifications too!`